### PR TITLE
NN-1754 Adding address details to the Personal page.

### PR DESCRIPTION
**Pull request by andrewrlee**
Added address details and parallelised the booking details view api calls.

Also fixed a few tests which weren't asserting anything.

### CODE REVIEW

File: app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import DisplayValue from '../../../../../components/FormComponents/DisplayValue'
+
+const InformationItem = ({ label, value, showIfMissing }) => {
```
Comment (andrewrlee) [inline app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: This name is not great - was considering `Detail` or `DetailItem`

Comment (andrewrlee) [inline app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: Not particularly happy with this name, any other suggestions would be welcome.

File: app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <div className="row border-bottom-line">
+      <div className="col-md-6 col-xs-6">
+        <span> {label} </span>
+      </div>
+      <div className="col-md-6 col-xs-6">
+        <strong>
+          {' '}
+          <DisplayValue value={value} />{' '}
```
Comment (jsrobertson) [inline app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: I reckon we could put the `<strong>` inside `<DisplayValue />`

File: app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import PropTypes from 'prop-types'
+import DisplayValue from '../../../../../components/FormComponents/DisplayValue'
+
+const InformationItem = ({ label, value, showIfMissing }) => {
+  if (!showIfMissing && !value) {
+    return null
+  }
+
+  return (
+    <div className="row border-bottom-line">
```
Comment (jsrobertson) [inline app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: Good opportunity to ditch this grid and use `<GridRow />` and `<GridCol />` from govuk-react.  Will come and help you.

File: app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js (hunk 1 above)
Comment (jsrobertson) [reply app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: `<ValueWithLabel />`?

Comment (jsrobertson) [reply app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: Nice work on getting rid of the repetition

Comment (andrewrlee) [reply app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: There was already a `ValueWithLabel` which was used for displaying "Assigned Staff Members" on quick look so I've merged the two components as they were doing the same thing..

File: app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js (hunk 2 above)
Comment (andrewrlee) [reply app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: No longer using DisplayValue

File: app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js (hunk 3 above)
Comment (andrewrlee) [reply app/containers/Bookings/Details/OffenderDetails/elements/InformationItem.js]: moved away from using old grid stuff with John's help

File: app/containers/Bookings/Details/OffenderDetails/index.js [hunk 4]
```diff
@@ -155,6 +104,32 @@ const OffenderDetails = ({ offenderDetails, showPhoto }) => {
         </div>
       </div>
 
+      <div className="row">
+        <div className="col-md-12">
+          <h3 className="heading-medium">Address Details</h3>
```
Comment (jsrobertson) [inline app/containers/Bookings/Details/OffenderDetails/index.js]: `import { H3 } from '@govuk-react/heading'`  and then use `<H3></H3>`.  You can pass in `size="MEDIUM"` if it needs it.

Comment (andrewrlee) [reply app/containers/Bookings/Details/OffenderDetails/index.js]: as discussed - there is some styling on this component which makes this a non-trivial fix as so out of scope for this PR.

**Pull request merged**